lint-staged.config.js

```diff
@@ -8,10 +8,15 @@ const eslintGlob = `*.{${configHelpers.extensions.join(',')}}`;
 module.exports = {
   // Run eslint in fix mode followed by prettier (must be done in sequence, separate from other
   // prettier formatting, since both commands can modify files)
-  [eslintGlob]: ['node ./scripts/lint-staged/eslint', 'prettier --write'],
+  // TODO: revert back to commented glob once web-components upgrades prettier
```

Comment:
can we create issue on this pls and add it to vbuild project or to WC folks to handle? thx

Reply:
FWIW, I'm tracking this on our feature branch. We're in the midst of a significant breaking change and rebasing against prettier changes would be chaos. @ecraig12345 was kind enough to drop this so that we can make the change on our feature branch without all that chaos :).

Reply:
looks good